src/xenia/gpu/vulkan/texture_cache.cc

```diff
@@ -404,13 +433,23 @@ TextureCache::Texture* TextureCache::DemandResolveTexture(
       },
       this, texture);
 
+  // Invalidate any textures that share this memory location
+  for (auto it = textures_.begin(); it != textures_.end(); ++it) {
```

Comment:
This block of code is causing a texture to be freed twice - it appears that an access watch fires and then this block of code runs over the texture as well.